src/Template/Element/sort_menu.ctp

```diff
@@ -0,0 +1,35 @@
+<md-menu md-offset="5 50" md-position-mode="target-right target">
+    <md-button ng-click="$mdOpenMenu($event)">
+        <md-icon>sort</md-icon> Sort by
```

Comment:
The string "Sort by" is part of the UI so it should be translatable. Please [put in into a `__()` call](https://book.cakephp.org/3/en/core-libraries/internationalization-and-localization.html).

Reply:
OK I am updating the code.